src/utils/__tests__/extentUtils.spec.js

```diff
@@ -0,0 +1,112 @@
+import { expect } from 'chai'
+import { describe, it } from 'vitest'
+
+import { LV95 } from '@/utils/coordinates/coordinateSystems'
+import { getExtentForProjection } from '@/utils/extentUtils'
+import { getKmlExtent } from '@/utils/kmlUtils.js'
+
+describe('Test extent utils', () => {
```

Comment:
Here we should also add edge test cases:

- Empty KML/GPX
- KML/GPX outside of projection bound

Or do you plan to do it in a separate PR?